### PR TITLE
Us 2 visitor navigation

### PR DESCRIPTION
**Pull request by the-color-bliu**
- [x] done

User Story 2, Visitor Navigation

As a visitor
I see a navigation bar
This navigation bar includes links for the following:
- [x] a link to return to the welcome / home page of the application ("/")
- [x] a link to browse all items for sale ("/items")
- [x] a link to see all merchants ("/merchants")
- [x] a link to my shopping cart ("/cart")
- [x] a link to log in ("/login")
- [x] a link to the user registration page ("/register")

- [x] Next to the shopping cart link I see a count of the items in my cart

### CODE REVIEW

**Review by helloeduardo — APPROVED** [review]
Thanks! Looks great!

**Comment by nmking22** [comment]
Merge approved!

File: config/routes.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
   get '/register', to: "users#new"
   post '/register', to: "users#create"
   get '/profile', to: 'users#show'
+  get '/login', to: 'sessions#new'
   #merchants
```
Comment (BJSherman80) [inline config/routes.rb:9]: we will hit this session controller at us13!!

File: spec/features/navigation_spec.rb [hunk 2]
```diff
@@ -25,13 +26,62 @@
...
+    end
+
+    it "I can see a login page indicator on all pages" do
+      visit '/'
+
+      within 'nav' do
+        expect(page).to have_content("Log In")
+      end
+      # click_link('Log In')
+      # expect(current_path).to eq('/sessions/new')
```
Comment (BJSherman80) [inline spec/features/navigation_spec.rb:62]: make sure we uncomment these at us13

**Pull request merged**